src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs

```diff
@@ -439,11 +442,13 @@ public async Task<IActionResult> PauseApprenticeship(PauseRequestViewModel viewM
             if (viewModel.PauseConfirmed.HasValue && viewModel.PauseConfirmed.Value)
             {
                 var pauseRequest = new PauseApprenticeshipRequest { ApprenticeshipId = viewModel.ApprenticeshipId };
-
+                
                 await _commitmentsApiClient.PauseApprenticeship(pauseRequest, CancellationToken.None);
+                
+                TempData.AddFlashMessage(ApprenticePausedMessage, ITempDataDictionaryExtensions.FlashMessageLevel.Info);

```

Comment:
Can you please add a flash message for stopped journey as well, as after stopping it may to go to other page instead of details.